com.github.you_win.VPupPr.yml

```diff
@@ -0,0 +1,50 @@
+id: com.github.you_win.VPupPr
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: OpenSeeFaceGD.x86_64
+finish-args:
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--share=network" # to set up the deps for openseeface
```

Comment:
if I read the comment right, you want this at build time?

1. this is not what this does
2. build is offline so it wouldn't work anyway.

The question is do you need network access at runtime?

Reply:
They go to ~/.var/app/com.github.you_win.VPupPr/data/godot/app_userdata/OpenSeeFaceGD/venv/ for the face tracking. Network access is also required if the user wants to use an external instance of the face tracker.

Reply:
The dependencies now get installed to ~/.var/app/com.github.virtual_puppet_project.vpuppr/data/godot/app_userdata/OpenSeeFaceGD/venv/